code/game/objects/items/bodybag.dm

```diff
@@ -43,3 +43,42 @@
 	w_class = WEIGHT_CLASS_SMALL
 	origin_tech = "bluespace=4;materials=4;plasmatech=4"
 
+/obj/item/bodybag/bluespace/examine(mob/user)
+	..()
+	if(contents.len)
+		user << "<span class='notice'>You can make out the shapes of [contents.len] objects through the fabric.</span>"
+
+/obj/item/bodybag/bluespace/Destroy()
+	for(var/atom/movable/A in contents)
+		A.forceMove(get_turf(src))
+		if(isliving(A))
+			A << "<span class='notice'>You suddenly feel the space around you torn apart! You're free!</span>"
+			var/mob/living/L = A
+			L.Weaken(3)
+	return ..()
+
+/obj/item/bodybag/bluespace/deploy_bodybag(mob/user, atom/location)
+	var/obj/structure/closet/body_bag/R = new unfoldedbag_path(location)
+	for(var/atom/movable/A in contents)
+		A.forceMove(R)
+		if(isliving(A))
+			A << "<span class='notice'>You suddenly feel air around you! You're free!</span>"
+			var/mob/living/L = A
+			L.Weaken(3)

```

Comment:
It can go either way. I mainly wanted to prevent people just bursting out of the bags and zipping around instantly, but at the same time that's possible for other lockers. What do you think?